packages/wonder-blocks-layout/components/media-layout.js

```diff
@@ -4,10 +4,25 @@ import type {StyleDeclaration} from "aphrodite";
 import type {StyleType} from "@khanacademy/wonder-blocks-core";
 
 import MediaLayoutContext from "./media-layout-context.js";
-import {VALID_MEDIA_SIZES} from "../util/specs.js";
 import type {MediaSize, MediaSpec} from "../util/types.js";
 import type {Context} from "./media-layout-context.js";
 
+const queries = [
+    // normal queries
+    "(max-width: 767px)", // small
+    "(min-width: 768px) and (max-width: 1023px)", // medium
+    "(min-width: 1024px)", // large
+
+    // internal tools
+    "(min-width: 1px)", // large
+
+    // modal queries
+    "(max-width: 767px)", // small
+    "(min-width: 768px)", // large
+];
+
+const MEDIA_QUERY_LISTS: {[key: string]: MediaQueryList} = {};
```

Comment:
Hmm - an all-caps constant implies to me that we expect this to be used elsewhere, but this really instant a constant, it's just a variable, so probably `const mediaQueryLists: ...` is fine.

Reply:
I usually think about whether a variable exists at the top level (as opposed to whether it's being exported) when deciding whether to use all-caps.

Reply:
That doesn't really explain why `queries` is lowercase above it though.  I'm not very consistent. 😅